### PR TITLE
Add `showNonLocalizedStrings` option to XCScheme+LaunchAction

### PR DESCRIPTION
**Pull request by ladislas**
Needed by https://github.com/tuist/tuist/pull/5782

Add `showNonLocalizedStrings` to UPPERCASE non localized strings.

- [x] add the missing property
- [x] implement init based on the other properties

### CODE REVIEW

File: Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift [hunk 1]
```diff
@@ -216,6 +219,11 @@ extension XCScheme {
 
             language = element.attributes["language"]
             region = element.attributes["region"]
+            if let elementShowNonLocalizedStrings = element.attributes["showNonLocalizedStrings"] {
+                self.showNonLocalizedStrings = elementShowNonLocalizedStrings == "YES"
+            } else {
+                self.showNonLocalizedStrings = false
+            }
```
Comment (danyf90) [inline Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift:226]: Isn't this enough?
```suggestion
            self.showNonLocalizedStrings = element.attributes["showNonLocalizedStrings"] == "YES"
```

**Review by danyf90 — APPROVED** [review]

File: Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift (hunk 1 above)
Comment (ladislas) [reply Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift]: yes! I've changed it :)

**Review by luispadron — APPROVED** [review]

**Comment by pepicrft** [comment]
@all-contributors add @ladislas for code

File: Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift [hunk 2]
```diff
@@ -327,6 +331,11 @@ extension XCScheme {
             if let region = region {
                 element.attributes["region"] = region
             }
+
+            if showNonLocalizedStrings {
```
[the remark this answers is not in the document]
Comment (ladislas) [reply Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift:335]: I've added the test, let me know if this what you had in mind or if I need to update it :)

**Review by pepicrft — APPROVED** [review]

**Pull request merged**